tests/mcp_server.py

```diff
@@ -16,4 +18,22 @@ async def celsius_to_fahrenheit(celsius: float) -> float:
     return (celsius * 9 / 5) + 32
 
 
-mcp.run()
+@mcp.tool()
+async def get_log_level(ctx: Context[ServerSession, None]) -> str:
+    """Get the current log level.
+
+    Returns:
+        The current log level.
+    """
+    await ctx.info('this is a log message')
```

Comment:
should there be a test of this?

Reply:
no I don't think so, we're just checking it doesn't cause the client to hang as it used to.